packages/clay-dropdown/src/ClayDropdownBase.soy

```diff
@@ -391,43 +367,78 @@
 	{@param? icon: string}
 	{@param? inputName: string}
 	{@param? inputValue: string}
-	{@param? isInList: bool}
 	{@param? items: list<?>}
 	{@param? itemsIconAlignment: string}
 	{@param? separator: bool}
 	{@param? spritemap: string}
 
-	{if $label != ''}
-		{if $isInList}
-			<li class="dropdown-subheader">
-				{$label}
-			</li>
-		{else}
-			<div class="dropdown-subheader">
-				{$label}
-			</div>
-		{/if}
-	{/if}
+	<div class="dropdown-subheader" role="presentation">
```

Comment:
hey @carloslancha, In cases of using `groups` the `$label` is **required** in soy but no error is charged, this ends up printing and leaving empty spaces.

<img width="273" alt="screen shot 2018-01-25 at 11 29 15" src="https://user-images.githubusercontent.com/13750819/35393389-9814ad74-01c3-11e8-9861-ab46b13be250.png">

Reply:
Hey @matuzalemsteles we can't throw any errors 'cause is a deltemplate. I'm adding a conditional anyway. Thx!